src/config/routes.js

```diff
@@ -6,10 +6,6 @@ export const buildPerformViewItemRoute = (id = ':id') =>
   `${GRAASP_PERFORM_HOST}/${id}`;
 export const HOME_ROUTE = '/';
 export const ALL_COLLECTION_ROUTE = '/all-collections';
```

Comment:
Since this exists there, why do you need to define it again? (supper/constants)

Reply:
I think it is because of `next.js`, I'm not able to import from `src/config` folder. If I try to do so, it will throw an error saying it could not destructure `getRuntimeConfig` from `getConfig()`. That's why I created the constants again under cypress folder